### PR TITLE
Comment escape char

### PR DESCRIPTION
**Pull request by vertein**
Before:
image

image

After:
image

image

### CODE REVIEW

**Comment by keirserrie** [comment]
:+1:

File: my-profile-middleware-impl/src/main/java/edu/wisc/my/profile/mapper/ContactInformationMapper.java [hunk 1]
```diff
@@ -145,7 +146,7 @@ private static JSONObject jsonifyEmergencyContact(ContactInformation eci) {
     JSONObject emergencyContact = new JSONObject();
     emergencyContact.put("EMERGENCY NAME", eci.getPreferredName());
     emergencyContact.put("RELATION", eci.getRelationship());
-    emergencyContact.put("RELATION COMMENT", eci.getComments());
+    emergencyContact.put("RELATION COMMENT", eci.getComments()+" ");
```
Comment (timlevett) [inline my-profile-middleware-impl/src/main/java/edu/wisc/my/profile/mapper/ContactInformationMapper.java]: Should we add in a "if ends in quote then add space"? Baby concern about saving over and over will add more and more spaces.

Comment (vertein) [inline my-profile-middleware-impl/src/main/java/edu/wisc/my/profile/mapper/ContactInformationMapper.java]: I had this thought too.  I was on the fence and could go either way.  Spaces vs computational logic.  Let's do it.  I'll update.

Comment (vertein) [inline my-profile-middleware-impl/src/main/java/edu/wisc/my/profile/mapper/ContactInformationMapper.java]: Checked and no longer will append unlimited spaces

**Comment by timlevett** [comment]
:+1:

**Comment by apetro** [comment]
:+1:

**Pull request merged**